functions/tests/platform/assistant/handler.spec.js

```diff
@@ -86,6 +89,28 @@ describe('platform', () => {
           expect(warning).to.have.been.called;
         });
       });
+
+      it('should gracefully prompt user in case of http error inside of a handler', () => {
+        actions.set('on-action', {
+          default: () => {
+            throw new errors.HTTPError();
+          },
+        });
+
+        const handler = handlerBuilder(actions);
+        handler(buildIntentRequest({
+          action: 'on-action',
+          lastSeen: null,
+        }), res);
+
+        return wait()
+          .then(() => {
+            expect(res.speech()).to.includes(
+              'We experience some technical problems on IA server, ' +
+              'please try again later or try something else.'
```

Comment:
Change this to `We are currently experiencing some technical difficulties on the Archive server. Please try again later or try saying something else.`